client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -40,9 +40,9 @@ export const CurrencyInput: FC<CurrencyInputProps> = ({
   value,
   ...restOfProps
 }) => {
-  const { c, options, language } = useCurrency();
-  const prefix = language !== 'fr' ? options.symbol : '';
-  const suffix = language === 'fr' ? options.symbol : '';
+  const { c, options, currencyCode } = useCurrency();
+  const prefix = currencyCode !== 'EUR' ? options.symbol : '';
+  const suffix = currencyCode === 'EUR' ? options.symbol : '';
```

Comment:
I hadn't remembered the hard coded values in here!
Could you make this more flexible please - I would do this:
```suggestion
  const { c, options } = useCurrency();
  const isSymbolLast = options.pattern.endsWith('!');
  const prefix = !isSymbolLast ? options.symbol : '';
  const suffix = isSymbolLast ? options.symbol : '';
```

which defaults to the symbol being a prefix, and is using the formatting pattern from the currency options, not a hard coded value.